frontends/PyCDE/src/pycde/system.py

```diff
@@ -89,6 +90,20 @@ def _get_ip(self):
   def set_debug():
     ir._GlobalDebug.flag = True
 
+  def import_hw_modules(self, path: Path):
```

Comment:
Why muck around with file I/O? Just accept a string then require the user to load from disk. More flexibility.

Reply:
I think actually the API should be just a list of modules returned from `module.import_hw_module`. This seems more orthogonal, and if you are going to actually instantiate the imported modules, you'll need to import them first before declaring modules and instantiating the system, and can just pass them to this API.